core/domain/skill_domain.py

```diff
@@ -910,6 +910,28 @@ def update_rubrics_from_model(cls, versioned_rubrics, current_version):
 
         versioned_rubrics['rubrics'] = updated_rubrics
 
+    def get_all_html_content_strings(self):
+        """Returns all the field values in the skill that has html content.
```

Comment:
"field values" is misleading -- you are actually returning the html strings themselves, not (e.g.) the misconceptions field.

Perhaps: Returns all html strings that are part of the skill (or any of its subcomponents).

Reply:
Done